README.md

```diff
@@ -9,16 +9,38 @@
 ![license](https://img.shields.io/github/license/streamdevs/webhook.svg)
 ![Node.js CI](https://github.com/streamdevs/webhook/workflows/Node.js%20CI/badge.svg)
 
-A webhook to forward GitHub events to StreamLabs Alerts and Twitch Chat.
+## Goal
 
-![Apr-21-2020 9-18-24 PM](https://user-images.githubusercontent.com/7255298/79904917-bc816e00-8415-11ea-98ca-7525b169ef49.gif)
+A webhook to forward GitHub and GitLab events to StreamLabs Alerts and Twitch Chat.
 
-## Deploy to Heroku
+This is a tool created with live-coding streamers in mind. The goal is to have another source of
+interaction and showcase relevant activity on GitHub to their audience on streaming platforms
+generating more interest in their Open Source projects.
+
+![StreamDevs Webhook Demo](https://user-images.githubusercontent.com/7255298/79904917-bc816e00-8415-11ea-98ca-7525b169ef49.gif)
+
+## Features
+
+### GitHub
```

Comment:
Should we add a `GitLab` section too?

Reply:
Moved into https://github.com/streamdevs/webhook/issues/180